weblate/addons/removal.py

```diff
@@ -59,7 +59,7 @@ class RemoveSuggestions(RemovalAddon):
     name = 'weblate.removal.suggestions'
     verbose = _('Remove old suggestions')
     description = _(
-        'Remove suggestions without votes older than given age in days.'
+        'Days to keep suggestions without votes.'
```

Comment:
This is long description of the addon, your text doesn't make sense there (samoe for description above):

![Screenshot_2019-05-11 Weblate Application](https://user-images.githubusercontent.com/212189/57571745-14e07b00-7412-11e9-92ea-ebd581efcb63.png)